### DISCUSSION TITLE
timelapse

### DISCUSSION THREAD

**Discussion by trinhpxl** [keiwando/toonsquid#8] [Ideas]
just wondering like how flipaclip and procreate has timelapse features for ur animations ! is it possible to ever import that feature into TS ? :3 if its not alr on ur long list of potential features XD

**Comment by keiwando** [comment]
Thanks for the suggestion! I wasn't yet planning on it and have now added it to the list, but it has a relatively low priority compared to most other feature requests at the moment.

**Reply by nicolasgallay** [reply]
Hello, do you have any updates about this feature ?

**Reply by starninjaart-cell** [reply]
Adding my vote on here.  Just made a new Github account to add my upvote.
